README.md

```diff
@@ -1,27 +1,49 @@
 # CognitoForms React
 
-A React component for embedding CognitoForms forms, making it easy to embed in a React-like way.
+---
 
-Usage:
+Easily embed CognitoForms in React.
+This package was forked from [@tylermenezes/cognitorforms-react](https://github.com/tylermenezes/cognitoforms-react)
+
+## Usage
+
+---
 
 ```jsx
-import CognitoForm from '@tylermenezes/cognitoforms-react';
-
-
-export default () => (
-  <>
-    <h1>My Demo Page</h1>
-    <CognitoForm
-      formId={61}
-      accountId={`SOMEACCOUNTIDSTRING`}
-      prefill={{
-        Name: {
-          First: 'John',
-          Last: 'Peter',
-        }
-      }}
-      css="* { color: red !important }"
-    />
-  </h1>
-);
+import CognitoForm from '@wpromote/cognitoforms-react';
+
+const MyComponent = () => {
+	return (
+		<>
+			<h1>My Demo Page</h1>
+			<CognitoForm
+				formId={61}
+				accountId={`SOMEACCOUNTIDSTRING`}
+				prefill={{
+					Name: {
+						First: 'John',
+						Last: 'Peter',
+					},
+				}}
+				css='* { color: red !important }'
+			/>
+		</>
+	);
+};
+
+export default MyComponent;
 ```
+
+## Install
```

Comment:
I don't know that we need this "Install" section. When we add this to our front end, we'll want to use a commit hash so we pin to a specific version so this might be misleading

Reply:
Ahh okay, was not sure if we needed to include anything else with it, makes sense though.